### ISSUE TITLE
unable to switch to S3

### ISSUE THREAD

**Issue by llopez** [spree/spree#7835]
I couldn't find the way to switch to s3 storage.

I have followed the [guides](http://guides.spreecommerce.org/developer/s3_storage.html) about using S3 with spree but it did not work.

I am getting:
AWS::S3::Errors::AccessDenied in Spree::Admin::ImagesController#create
even if the bucket is public. very strange issue.

I am running Rails 4.2.8 + Ruby 2.4.0

I am using aws-sdk-v1 gem given that spree does not support v2

Please help!

**Comment by nnande** [comment]
Hello @Ilopez
What you need to do is to grant your AWS user write object access to your s3 bucket.

To accomplish that perform these steps:

1. Go to the Amazon S3  console and click your bucket name.
2. Go to **Permissions -> Access Control List** tab
3. Under the 'Manage users' section go to your user's permissions by clicking it and make sure Object write access is checked.

Users are identified by Canonical User ID by default. You can find in your account settings. 

Basically, It should do the job.
